src/bin/vip-export-sql.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import command from '../lib/cli/command';
+import { ExportSQLCommand } from '../commands/export-sql';
+import { makeCommandTracker } from '../lib/tracker';
+
+
+const examples = [
+  {
+    usage: 'vip export sql @mysite.develop --output=/home/user/export.sql.gz',
```

Comment:
Can we add another example without the `--output` flag? 

https://a8c.slack.com/archives/CLKJLBFB8/p1684941954409949?thread_ts=1684941892.014789&cid=CLKJLBFB8

Reply:
nevermind, I think this would make sense if we support stdout